tests/dummy/app/styles/app.scss

```diff
@@ -1,10 +1,10 @@
-@import 'variables/variables';
+@use 'variables/_variables' as *;
 
-@import 'components/components';
-@import 'object-patterns/object-patterns';
-@import 'pages/pages';
-@import 'typography/typography';
-@import 'utilities/utilities';
+@use 'components/components';
+@use 'object-patterns/object-patterns';
+@use 'pages/pages';
+@use 'typography/typography';
+@use 'utilities/utilities';
 
 html {
```

Comment:
[suggestion] not related to this PR, but I would suggest to move the global declarations in a dedicated file (eg. `global.scss` and import (`@use`) it as first item after the variables, before the other files (components, patterns, etc).

Reply:
Thank you for the suggestion! I've captured it in a separate issue: https://github.com/ember-a11y/ember-a11y-testing/issues/563